src/features/advanced/exp-expert-eta.ts

```
@@ -0,0 +1,168 @@
+      };
+    }
+  }
+
+  // Otherwise, estimate the remaining number of days.
+  let experiencePerMs = 0;
+  for (const line of lines) {
+    const recurringOrders = getRecurringOrders(line);
+    if (recurringOrders.length === 0) {
+      continue;
+    }
+    const totalDuration = sumBy(recurringOrders, x => x.duration?.millis ?? Infinity);
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)